tests/test_vortex_2d.py

```diff
@@ -1,25 +1,132 @@
 """> PyDRex: tests for CPO stability in 2D vortex and Stokes cell flows."""
 
+import contextlib as cl
 import functools as ft
 import sys
 
 import numpy as np
 import pytest
+from numpy import testing as nt
 
 from pydrex import core as _core
 from pydrex import diagnostics as _diagnostics
 from pydrex import io as _io
 from pydrex import logger as _log
 from pydrex import minerals as _minerals
 from pydrex import pathlines as _path
+from pydrex import stats as _stats
 from pydrex import utils as _utils
 from pydrex import velocity as _velocity
 from pydrex import visualisation as _vis
+from pydrex import geometry as _geo
 
 Pool, HAS_RAY = _utils.import_proc_pool()
 
-# Subdirectory of `outdir` used to store outputs from these tests.
 SUBDIR = "2d_vortex"
+"""Subdirectory of `outdir` used to store outputs from these tests."""
```

Comment:
Triple quotes?

Reply:
So that the text appears in the generated html